### PR TITLE
Adds default span classes to visible and audible messages

### PR DESCRIPTION
**Pull request by kingofkosmos**
## About The Pull Request
General consensus seemed to be that separating talking-messages from other action-messages seen in the chat-window would make it less cluttered.

And because talking-messages are just plain black text, action-messages should have some other color. The 'notice' span class seemed to be appropriate for action-messages which didn't have any span class, so they are not alarming messages or combat messages or anything important, just plain "you see someone wrenching the girder" etc.

So instead of adding notice-spans to over 500 visible_messages, I thought they could all be defaulted to 'notice', unless defined otherwise (I made sure that later defined span classes override the default one).

Audible messages have the default set to 'italics', which I made the same color as 'notice'.

TL;DR This wraps the visible messages to span class 'notice', and with audible messages default to span class 'italics'. Also changes 'italics' to the same color as 'notice'.

## Why It's Good For The Game
Separates talking-messages from everything else that's going on, making the chat screen more readable.

### CODE REVIEW

File: code/modules/mob/mob.dm [hunk 1]
```diff
@@ -198,21 +198,29 @@
 			continue
 		if(M in ignored_mobs)
 			continue
-		var/msg = message
+		var/msg = "<span class='notice'>"
```
Comment (actioninja) [inline code/modules/mob/mob.dm]: concat is slow in byond, use formatting where ever possible

`var/msg = "<span class='notice'>[message]</span>"`

File: code/modules/mob/mob.dm [hunk 2]
```diff
@@ -198,21 +198,29 @@
...
+		msg += message
+		msg += "</span>"
 		if(M == src) //the src always see the main message or self message
 			if(self_message)
-				msg = self_message
+				msg = "<span class='notice'>"
```
Comment (actioninja) [inline code/modules/mob/mob.dm]: ditto

File: code/modules/mob/mob.dm [hunk 3]
```diff
@@ -198,21 +198,29 @@
...
+				msg += self_message
+				msg += "</span>"
 		else
 			if(M.see_invisible<invisibility || (T != loc && T != src))//if src is invisible to us or is inside something (and isn't a turf),
 				if(blind_message) // then people see blind message if there is one, otherwise nothing.
-					msg = blind_message
+					msg = "<span class='italics'>"
```
Comment (actioninja) [inline code/modules/mob/mob.dm]: ditto

File: code/modules/mob/mob.dm [hunk 4]
```diff
@@ -198,21 +198,29 @@
...
+					msg += blind_message
+					msg += "</span>"
 				else
 					continue
 
 			else if(T.lighting_object)
 				if(T.lighting_object.invisibility <= M.see_invisible && T.is_softly_lit()) //the light object is dark and not invisible to us
 					if(blind_message)
-						msg = blind_message
+						msg = "<span class='italics'>"
```
Comment (actioninja) [inline code/modules/mob/mob.dm]: ditto

File: code/modules/mob/mob.dm [hunk 5]
```diff
@@ -234,10 +242,14 @@
 	if(hearing_distance)
 		range = hearing_distance
 	for(var/mob/M in get_hearers_in_view(range, src))
-		var/msg = message
+		var/msg = "<span class='italics'>"
```
Comment (actioninja) [inline code/modules/mob/mob.dm]: ditto

File: code/modules/mob/mob.dm [hunk 6]
```diff
@@ -234,10 +242,14 @@
...
+		msg += message
+		msg += "</span>"
 		if(self_message && M==src)
-			msg = self_message
-		M.show_message( msg, 2, deaf_message, 1)
+			msg = "<span class='italics'>"
```
Comment (actioninja) [inline code/modules/mob/mob.dm]: ditto

File: code/modules/mob/mob.dm [hunk 7]
```diff
@@ -254,7 +266,10 @@
 	if(hearing_distance)
 		range = hearing_distance
 	for(var/mob/M in get_hearers_in_view(range, src))
-		M.show_message( message, 2, deaf_message, 1)
+		var/msg = "<span class='italics'>"
```
Comment (actioninja) [inline code/modules/mob/mob.dm]: ditto

**Review by actioninja — CHANGES_REQUESTED** [review]
I'm not entirely sure if this is the best method to go about doing this, I'll have to ask around a bit. In the mean time here's some simple code cleanup.

File: code/modules/mob/mob.dm (hunk 1 above)
Comment (kingofkosmos) [reply code/modules/mob/mob.dm]: I didn't know it could be that easy, thanks!

**Comment by kingofkosmos** [comment]
> I'm not entirely sure if this is the best method to go about doing this, I'll have to ask around a bit. In the mean time here's some simple code cleanup.

I appreciate you taking the time to check things out. Atleast everything looked fine when I tested if it works. Tried adding the same span-wrap for to_chat as well, but realized that's used for way too many other things! But I'm feeling pretty safe about these procs.

**Comment by actioninja** [comment]
So after a bit more thought and discussion, this isn't the right way to be going about this. The right way would be closer to how `say` does it, and show_message and visible_message would both need some pretty heavy refactors in order for that approach to work. If you want to try to take a shot at that go ahead though.

**Comment by kingofkosmos** [comment]
What makes it the wrong way? Does it take unnecessary resources? And if so, how would it differ if I went through all the 500+ visible_messages and added span class='notice' to all of them?
